### DISCUSSION TITLE
zksync deployment

### DISCUSSION THREAD

**Discussion by Ptyline** [Cyfrin/foundry-full-course-cu#3088] [General]
hello buddies, I've been trying to deploy to zksync and I  have been getting error. I also tried using the latest version v0.7.1 yet not working. Please I need help

**Comment by Ptyline** [comment]
![Screenshot (4)](https://github.com/user-attachments/assets/65c2e75b-35f4-4a87-a340-d2fd0d60347f)

**Comment by Ptyline** [comment]
Hey buddies....I've solved it. 
I watched the preceding video to debug it

**Reply by EngrPips** [reply]
Hey @Ptyline. what did you do to solve it?

**Reply by Ptyline** [reply]
I created a new folder, copied the .sol file into the new folder, refreshed my pc then deployed....voilaa!!!

**Reply by EngrPips** [reply]
> I created a new folder, copied the .sol file into the new folder, refreshed my pc then deployed....voilaa!!!

Great heads-up. Thanks buddy.

**Reply by siddharthseth888** [reply]
is it necessary to get The simplestorage1.sol contract must be in contracts folder for successful compilation?
I am not able to compile my contract file can you help me?

**Reply by EngrPips** [reply]
Yeah, please put your contract file inside the `contracts` folder, refresh your Remix, and then retry the compilation.

**Reply by siddharthseth888** [reply]
Ok 'll try..

**Reply by Ptyline** [reply]
> is it necessary to get The simplestorage1.sol contract must be in contracts folder for successful compilation? I am not able to compile my contract file can you help me?

Yes, create a new folder then copy the file into the new folder....I hope it works for you

**Reply by EngrPips** [reply]
Yeah
